src/routes/download/drs.service.ts

```diff
@@ -0,0 +1,70 @@
+import urlJoin from 'url-join';
+import fetch from 'node-fetch';
+
+import { AppConfig } from '@/config/global';
+import logger from '@/logger';
+
+// types from https://ga4gh.github.io/data-repository-service-schemas/preview/release/drs-1.0.0/docs/#_accessmethod
+// will only be using https
+enum AccessType {
+  HTTPS = 'https',
+  S3 = 's3',
+  GS = 'gs',
+  FTP = 'ftp',
+  FILE = 'file',
+  GSIFTP = 'gsiftp',
+  GLOBUS = 'globus',
+  HTSGET = 'htsget',
+}
+
+interface AccessMethod {
+  type: AccessType;
+  access_url: { url: string };
+  region: string;
+}
```

Comment:
This naming confused me and I had to remove a bunch of comments I left. It might help if we rename these to be like: `DrsProtocols` and `AccessUrl`

Reply:
yeah that does sound better, will do!

Reply:
naming is hard. easier to do when just reviewing ;D